x-pack/plugins/security_solution/public/timelines/components/flyout/add_timeline_button/index.tsx

```diff
@@ -41,8 +41,8 @@ const AddTimelineButtonComponent: React.FC<AddTimelineButtonComponentProps> = ({
         className={ADD_TIMELINE_BUTTON_CLASS_NAME}
         data-test-subj="settings-plus-in-circle"
```

Comment:
If you end up updating the PR, mind giving this a more timeline related `data-test-subj`? Something like `new-timeline-header-action` or something specific to timeline that doesn't conflict with the existing `new-timeline-action`? No worries, if you don't make any additional changes

Reply:
Sure @michaelolo24 , I will check the impact on test, if it is not that big. I will go ahead and change it.

Reply:
@michaelolo24  What do you think about `timeline-create-open-control`?

Reply:
Works perfectly, thanks!